pkg/network/protocols/http/monitor.go

```diff
@@ -183,6 +194,7 @@ func (m *Monitor) Stop() {
 		return
 	}
 
+	m.processMonitor.Stop()
```

Comment:
same here, unless you change the GetProcessMonitor behavior, this can potentially null deref